types/builtins/aggregates.d.ts

```diff
@@ -0,0 +1,11 @@
+declare const _exports: {
```

Comment:
It it standard practice to commit the generated files? Or should we be adding `types/` to .gitignore and only generating them when building the module?

Reply:
Nope! Just wanted to demonstate the generated types.